sopel_waifu/waifu.json

```
@@ -1468,47 +1470,106 @@
+        "Inami Mahiru",
+        "Shirafuji Kyouko",
+        "Todoroki Yachiyo",
+        "Yamada Aoi",
+        "Takanashi Izumi",
+        "Takanashi Kazue",
+        "Takanashi Kozue",
+        "Takanashi Nazuna",
+        "Matsumoto Maya",
+        "Mashiba Mitsuki",
+        "Otoo Haruna",
+        "Kitahara Ritsuko",
+        "Suzutani Momoka"
     ],
     "Yahari Ore no Seishun Love Comedy wa Machigatteiru": [
-        "Yui Yuigahama",
-        "Yukino Yukinoshita",
-        "Komachi Hikigaya"
+        "Yukinoshita Yukino",
+        "Yuigahama Yui",
+        "Isshiki Iroha",
+        "Hikigaya Komachi",
+        "Hiratsuka Shizuka",
+        "Kawasaki Saki",
+        "Yukinoshita Haruno",
+        "Miura Yumiko",
+        "Ebina Hina",
+        "Tsurumi Rumi",
+        "Shiromeguri Meguri",
+        "Orimoto Kaori",
+        "Nakamachi Chika",
+        "Kawasaki Keika",
+        "Yuka",
+        "Hitomi",
+        "Mori",
+        "Yokko",
+        "Sagami Minami"
     ],
     "Yokohama Kaidashi Kikou": [
-        "Hatsuseno Alpha"
+        "Hatsuseno Alpha",
+        "Takatsu Kokone"
     ],
     "Yosuga no Sora": [
-        "Kasugano Sora"
+        "Amatsume Akira",
+        "Yorihime Nao",
+        "Migiwa Kazuha",
+        "Kasugano Sora",
+        "Nogisaka Motoka",
+        "Kuranaga Kozue",
+        "Ifukube Yahiro"
     ],
     "Yuru Yuri": [
         "Toshinou Kyouko",
-        "Akaza \"Who?\" Akari",
+        "Akaza Akari",
         "Funami Yui",
         "Yoshikawa Chinatsu",
         "Sugiura Ayano",
-        "Ikeda Chitose"
+        "Ikeda Chitose",
+        "Oomuro Sakurako",
+        "Furutani Himawari",
+        "Mirakurun",
+        "Rivarun",
+        "Furutani Kaede",
+        "Ikeda Chizuru",
+        "Nishigaki Nana",
+        "Matsumoto Rise",
+        "Funami Mari",
+        "Takaoka Hiro",
+        "Yoshikawa Tomoko",
+        "Oomuro Nadeshiko",
+        "Oomuro Hanako",
+        "Azuma Shihoko",
+        "Kitamiya Hatsumi",
+        "Minamino Tsubasa"
     ],
     "Yuyushiki": [
-        "Yui Ichii",
-        "Yuzuko Nonohara",
-        "Yukari Hinata"
+        "Nonohara Yuzuko",
+        "Ichii Yui",
+        "Hinata Yukari",
+        "Aikawa Chiho",
+        "Okano Kei",
+        "Hasegawa Fumi",
+        "Matsumoto Yoriko"
     ],
     "Zero no Tsukaima": [
-        "Louise",
+        "Louise Françoise Le Blanc de La Vallière",
+        "Charlotte Helene Orléans",
+        "Kirche Augusta Frederica von Anhalt-Zerbst",
         "Siesta",
-        "Tabitha",
-        "Tiffania Westwood",
+        "Montmorency Margarita la Fère de Montmorency",
         "Henrietta de Tristain",
-        "Kirche"
+        "Agnes Chevalier de Milan",
+        "Tiffania Westwood",
+        "Eléonore Albertine Le Blanc de La Blois de La Vallière",
+        "Cattleya Yvette La Baume Le Blanc de La Fontaine",
+        "Sheffield",
+        "Beatrice Yvonne von Guldenhorf",
+        "Irukukwu",
+        "Jessica",
+        "Chevreuse",
```

```suggestion
        "Chevreuse the Red Moon",
```

Or:

```suggestion
        "Professor Chevreuse",
```

Note she's pretty old, might not want to include.